test/mix/tasks/kitto.new_test.exs

```diff
@@ -0,0 +1,63 @@
+Code.require_file "../../../installer/lib/kitto_new.ex", __DIR__
```

Comment:
Do nothing at the context of this PR. After merging it, I'll probably extract the installer to a separate repo.